services/motion/builtin/move_request.go

```diff
@@ -61,8 +60,7 @@ const (
 type moveRequest struct {
 	requestType requestType
 	// geoPoseOrigin is only set if requestType == requestTypeMoveOnGlobe
-	geoPoseOrigin spatialmath.GeoPose
-	// poseOrigin is only set if requestType == requestTypeMoveOnMap
+	geoPoseOrigin     *spatialmath.GeoPose
 	poseOrigin        spatialmath.Pose
```

Comment:
Should Pose be a pointer too (my assumption is they fulfil the same purpose

Reply:
It is already a pointer since interfaces are definitionally pointers